app/assets/stylesheets/application.scss

```diff
@@ -1,12 +1,112 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 @import "mixins";
+@import 'https://fonts.googleapis.com/css?family=Balthazar';
 
 /* Global */
 
 hr {
   margin-top: 0;
   margin-bottom: 0;
+  border: 0;
+  height: 1px;
+  background-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0));
+}
+
+body {
+  background: url('/assets/concrete.png');
+  font-family: 'Balthazar', serif;
+}
+
+.clr {
+  clear: both;
+}
+
+/*FOOTER*/
+
+.Site {
+  display: flex;
+  min-height: 100vh;
+  flex-direction: column;
+}
+
+.Site-content {
+  flex: 1;
+}
+
+footer {
+  background-color: #979A9C;
+  height: 70px;
+}
+
+.footer_links li {
+  list-style-type: none;
+}
+
+.footer_links li a {
+  color: white;
+  text-decoration: none;
+}
+
+/* INDEX */
+
+.navbar-inverse {
+  background-color: #979A9C;
+  border: none;
+  box-shadow: 3px 3px 3px;
+}
+
+.navbar-header .navbar-brand {
+  color: white;
+  text-decoration: none;
+  font-size: 28px;
+  font-weight: bold; 
+}
+
+ul.nav.navbar-nav.navbar-right li a {
+  color: white;
+  padding-top: 15px;
+}
+
+.nav .navbar-nav .navbar-right {
+  margin-right: 50px;
+}
+
+.login {
+  width: 100%;
+}
+
+.remember {
+  padding-left: 10px;
+}
+
+.form-inputs input {
+  width: 100%;
+}
+
+.git_icon {
+  padding-left: 15%;
+}
+
+.link_icon {
```

Comment:
Not a big deal, but FYI this breaks CSS naming conventions. 
s/link_icon/link-icon